index.css

```diff
@@ -96,3 +102,78 @@ main {
     margin: auto;
     text-align: center;
 }
+
+
+.wrapper {
+    margin: 0 auto;
+    background-color: #ffffff;
+    height: 50px;
+    padding: 0;
+
+
+
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+    border-radius: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: #ffffff;
+
+
+}
```

Comment:
I would suggest header already being the wrapper for the header content and change the parameters:

header {
    width: 100%;
    height: 120px;
    padding: 0 30px;
    display: flex;
    position: fixed;
    align-items: center;
    justify-content: space-between;
    background-color: #ffffff;
}